src/js/common/schemaform/FormApp.jsx

```diff
@@ -34,7 +34,20 @@ moment.updateLocale('en', {
       return 'a.m.';
     }
     return 'p.m.';
-  }
+  },
+  monthsShort: [
+    'Jan.',
+    'Feb.',
+    'Mar.',
+    'Apr.',
+    'May',
+    'Jun.',
+    'Jul.',
+    'Sept.',
+    'Oct.',
+    'Nov.',
+    'Dec.'
+  ]
```

Comment:
Thanks for pointing out that Sep should be Sept. @peggygannon, are these updated month abbreviations all correct?